.travis_scripts/cross-test.sh

```diff
@@ -5,13 +5,13 @@ set -x
 
 # If no special target provided, use default cargo arch for building and run all unit tests
 if [ -z "$TARGET" ]; then
-    IRONCORE_ENV=dev cargo t --verbose
+    IRONCORE_ENV=dev cargo t --features "blocking" --verbose
```

Comment:
Just an optics suggestion, but would it be better to name this "synchronous" instead of "blocking"? The latter seems like it has a more negative connotation for consumers of this SDK.

Reply:
I'm open to naming it something like "synchronous" or "sync". We just chose this name because it matches the flag in `reqwest`, so we sort of accepted it like a standard.

Reply:
I am the one that suggested patterning after reqwest with the blocking terminology. It's certainly open for discussion, but I don't find blocking more negative then sync.